.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,2 @@
+<!-- Love streamlink? Please consider supporting our collective:
+👉  https://opencollective.com/streamlink/donate -->
```

Comment:
It's the `:point_right:` emoji.

Btw, we already have an [issue template in the root folder](https://github.com/streamlink/streamlink/blob/master/ISSUE_TEMPLATE.md). This one's in the `.github` folder and would override the main one.

Using an HTML comment is an elegant solution for not including text in any rendered markdown. But is it necessary here? People commenting on already exising issues may have some interest in supporting, too (without reading the readme).